src/xdocs/checks/blocks/rightcurly.xml

```diff
@@ -113,43 +113,46 @@
         </source>
         <p id="Example1-code">Example:</p>
         <source>
-public class Test {
+public class Example1 {
 
   public void test() {
 
+    boolean foo = false;
     if (foo) {
       bar();
-    }           // violation, right curly must be in the same line as the 'else' keyword
+    } // violation, ''}' at column 5 should be on the same line*'
     else {
       bar();
     }
 
     if (foo) {
       bar();
-    } else {     // OK
+    } else {
       bar();
     }
 
-    if (foo) { bar(); } int i = 0;  // violation
-                  // ^^^ statement is not allowed on same line after curly right brace
+    if (foo) { bar(); } int i = 0; // violation, ''}' at column 23 should be*'
```

Comment:
> violation, ''}' at column 23 should be*'

main part is hidden by *, lets remove `''}' at column 23` instead, in this line only one `}` so it is obvious where is violation.
please restore `// ^^^ statement is not allowed on same line after curly right brace` it will help users

Reply:
added actual violation msg as it was not accepting customize one.